src/main/java/org/quantumbadger/redreader/common/PrefsUtility.java

```diff
@@ -865,14 +867,52 @@ public static boolean network_tor(final Context context, final SharedPreferences
 		return result;
 	}
 
-	public static EnumSet<OptionsMenuUtility.OptionsMenuItemsPref> pref_menus_optionsmenu_items(final Context context, final SharedPreferences sharedPreferences) {
+	private static class AppbarItemInfo {
+		final Enum<OptionsMenuUtility.AppbarItemsPref> itemPref;
```

Comment:
I don't think this needs to be wrapped in `Enum<>`, you can just write the name of the enum :)

Reply:
(same for all usages elsewhere, e.g. line 874)

Reply:
Thank you for the tip, I actually realized this myself earlier and removed the wrappings.